src/app/collective-rewards/rewards/builders/AllTimeShare.tsx

```diff
@@ -57,7 +57,15 @@ export const AllTimeShare: FC<AllTimeShareProps> = ({ gauge, gauges, tokens: { r
   return (
     <MetricsCardWithSpinner isLoading={isLoading} borderless>
       <div className="flex flex-col gap-y-[10px]">
-        <MetricsCardTitle title="All time share" data-testid="AllTimeShare" />
+        <MetricsCardTitle
+          title="All time share"
+          data-testid="AllTimeShare"
+          tooltip={{
+            // eslint-disable-next-line quotes
+            text: "Total sum of your rewards, share from all the Builders' rewards",
```

Comment:
Can this sentence be a bit confusing?
wdyt:
Sum of the shares your earned from all the Builders' rewards